contracts/exchange-libs/scripts/generate-exchange-selectors.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
```

Comment:
Why is this in JS? Scripts can also be written in TS.

Reply:
My logic was that it's awkward to build a package in order to run a script that generates code for the package that has to be rebuilt again. Thinking about it, maybe a better place for the TS version of this script is `monorepo-scripts`?